sparse_strips/vello_bench/src/data.rs

```diff
@@ -81,7 +81,7 @@ impl DataItem {
 
         for path in &self.fills {
             flatten::fill(
-                Level::new(),
+                Level::try_detect().unwrap_or(Level::fallback()),
```

Comment:
This seems weird to me. I can see two reasonable options here:
1) We want to benchmark with the fallback level, in which case we should be passing it in directly
2) This is just an attempt to get the best possible level, in which case it should probably still use `Level::new`

(This comment is only about these calls in the context of `vello_bench`)